### PR TITLE
feat: GH-139 Add `build` section to docker-compose.yml

### PR DESCRIPTION
**Pull request by primeapple**

### CODE REVIEW

**Comment by madiele** [comment]
Sorry if I'm taking so long on this PR, I want to test how the docker compose up command works with this section, I remember in the past that this section would sometime make the up command do a build even when an image was present to download.
Once I can do some test I will come back to it

**Comment by primeapple** [comment]
Sure thanks a lot for your feedback! Take your time. If you don't want this change I'm fine with it as well.

File: docker-compose.yml [hunk 1]
```diff
@@ -18,6 +18,9 @@ services:
     #change "latest" to "X.X.X" to pin a version es: "1.0.4" will force the image to use to version 1.0.4, if you do please watch the repo for updates (tutorial in README.md)
     #change "latest" to "beta" if you want to test yet unreleased fixes/features (expect bugs and broken builds from time to time)
     image: madiele/vod2pod-rss:latest 
+    build:
+      dockerfile: ./Dockerfile
+      context: https://github.com/madiele/vod2pod-rss.git
```
Comment (madiele) [inline docker-compose.yml:23]:
```suggestion
    # uncomment to build vod2pod from scratch, only do this if your architecture is not supported
    #build: 
    #  dockerfile: ./Dockerfile
    #  context: https://github.com/madiele/vod2pod-rss.git
```

**Comment by madiele** [comment]
after some testing yeah, it's to random on what docker compose will do, apply the suggestion and then I will merge it ✌️

**Pull request merged**